mevislab.github.io/content/tutorials/basicmechanisms.md

```diff
@@ -157,35 +153,35 @@ The connection is drawn as a thin grey arrow between both modules with the arrow
+We will now establish a connection from the module `SyncFloat` to the second viewer, `Viewer2D1`. In order to do that, open the automatic panel `View2D1`. Draw a connection from the label `Float2` of the panel of the module `SyncFloat` to the label *startSlice* in the automatic panel of the module `View2D1`. Lastly, implement a connection between the parameter fields *startSlice* of both viewers. Draw the connection from `View2D1` to `View2D`.
 
 ![Synchronize both directions](/images/tutorials/basicmechanics/BM_15.png "Synchronize both directions")
 
 As a result, scrolling through the slices with the mouse wheel {{< mousebutton "middle" >}} in one of the viewers synchronizes the rendered slice in the second viewer. In this case, you can inspect the differences between smoothed and unsmoothed data on every single slice.
 
 ![Your final Network](/images/tutorials/basicmechanics/BM_16.png "Your final Network")
 
-You can also use the pre-defined Module `SynchroView2D` to reach the same goal (shown in [this chapter](/tutorials/visualization/visualizationexample1/) ).
+It is also possible to use the pre-defined module 'SynchroView2D' to accomplish a similar result.(`SynchroView2D`s usage is futherly described in [this chapter](/tutorials/visualization/visualizationexample1/) ).
 
 ### Grouping Modules {#TutorialGroupingModules}
 
-In this chapter we like to create a contour filter. To finalize the filter, add the modules `Arithmetic2` and `Morphology` to your workspace and connect the modules as shown below. Double-click the module `Arithmetic2` to open its panel. Change the field *Function* of the module `Arithmetic2` to use the function *subtract* in the panel of the module. With this, we finished our contour filter. You can inspect each processing step using the Output Inspector by clicking on the input and output connectors of the respective modules. The final results can be inspected using the viewers. If necessary, adjust the contrast by pressing the right-key and moving the cursor.
+A contour-filter can be created based on our previously created network. To finalize the filter, add the modules `Arithmetic2` and `Morphology` to your workspace and connect the modules as shown below. Double-click the module `Arithmetic2` to open its panel. Change the field *Function* of the module `Arithmetic2` to use the function *subtract* in the panel of the module. The contour filter is done now. You can inspect each processing step using the Output Inspector by clicking on the input and output connectors of the respective modules. The final results can be displayed using the viewer modules. If necessary, adjust the contrast by pressing the right-key and moving the cursor.
```

Comment:
contour filter

Reply:
Let's find rules on whether a specific word should be written in lower or upper case. I found it quite hard to distinguish between MeVisLab specific components (supposed to be written in upper case) and general vocab (usually lower case). 
e.g. Network/network, Module/module, and so on.

Reply:
Yes, I agree. It is currently inconsistent, though I think there are way too many capitalizations. In the existing MeVisLab documentation, all these names (e.g. module, network, macro module, viewer etc.) are lower-case (except in captions), and I tend to agree with this spelling. 

(note that this specific comment was about the hyphen, not the capitalization).